src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -824,10 +827,11 @@ export function Positions({
 }) {
   const { currency } = useCurrency();
   const { ready, params, singleAddressNormalized } = useAddressParams();
+  const addrIsSolana = isSolanaAddress(singleAddressNormalized);
   const { data, ...portfolioQuery } = useWalletPortfolio(
     { addresses: [params.address], currency },
     { source: useHttpClientSource() },
-    { enabled: ready }
+    { enabled: ready && !addrIsSolana }
```

Comment:
Probably, we will remove this when backend supports Solana?

Reply:
Yes